### PR TITLE
Rendering 4 components client-side

### PR DESCRIPTION
**Pull request by tmkuba**
https://trello.com/c/4XgQthSx

### CODE REVIEW

File: dist/template.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<html>
+  <head>
```
Comment (matgbx) [inline dist/template.html:2]: what is the difference between this file a dist/index.html?

**Review by matgbx — APPROVED** [review]
just a short comment for my own understanding but otherwise looks great

File: dist/template.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <div id="reviews"></div>
+
+    <script type="text/javascript" src="about-bundle.js"></script>
+    <script type="text/javascript" src="res-bundle-prod.js"></script>
+    <script type="text/javascript" src="menu-bundle.js"></script>
+    <script type="text/javascript" src="reviews-bundle.js"></script>
+    <script type="text/javascript">
+
+ReactDOM.render(
+  React.createElement(SummaryView, {id: <%= id %>}, null),
```
Comment (leungnamson1023) [inline dist/template.html:27]: neat trick with templating

File: server.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const express = require('express');
+const _ = require('underscore');
+
+const app = express();
+
+app.use(express.static('./dist'));
+
+// load the template file
+const compiled = _.template(fs.readFileSync('./dist/template.html', 'utf-8'));
```
Comment (leungnamson1023) [inline server.js:10]: nice

File: copy-bundles.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#/bin/sh
```
Comment (leungnamson1023) [inline copy-bundles.sh:1]: is this .sh file a "script" to copy files from one folder to another? and also, is there a script that runs through each individuals repo to build their webpack prod config and copy it to your destination folder?

**Review by leungnamson1023 — APPROVED** [review]
everything else looks good

File: copy-bundles.sh (hunk 4 above)
Comment (tmkuba) [reply copy-bundles.sh:1]: yeah, it's just a script to copy the files.  I haven't written a script to build everyone's webpack prod yet, though...

File: dist/template.html (hunk 1 above)
Comment (tmkuba) [reply dist/template.html:2]: dist/index.html is just the default file that is loaded (with restaurant id=305) when the client requests '/'.  However, template.html is the file that is used by the server to inject the requested restaurant id (GET '/3267') into the HTML file (and subsequently to the React components) to return to the client.

File: copy-bundles.sh (hunk 4 above)
Comment (leungnamson1023) [reply copy-bundles.sh:1]: gotcha!

**Pull request merged**